### PR TITLE
Fix: e2e

### LINKED ISSUE #109: ci: fix cypress e2e tests
Check [this](https://github.com/ubiquity/work.ubq.fi/actions/runs/11216977982/job/31177473349) failing cypress e2e testing workflow.

As far as I understand it fails because now `work.ubq.fi` [fetches](https://github.com/ubiquity/work.ubq.fi/blob/23d04fb764b63e8445453e2e9ff44b84afbce624/src/home/fetch-github/fetch-issues-full.ts#L5) github issues from the [devpool-issues.json](https://github.com/ubiquity/devpool-directory/blob/79330a6d55e22543a3bb102817e5075cb4dc4d5b/devpool-issues.json) while e2e tests [mock](https://github.com/ubiquity/work.ubq.fi/blob/23d04fb764b63e8445453e2e9ff44b84afbce624/cypress/e2e/devpool.cy.ts#L18-L23) regular github API calls.

What should be done:
- refactor [e2e tests](https://github.com/ubiquity/work.ubq.fi/blob/23d04fb764b63e8445453e2e9ff44b84afbce624/cypress/e2e/devpool.cy.ts) according to the current logic

### PR DESCRIPTION
**Pull request by nick-dev123**
Resolves #109 

Removes obsolete tests:
- Should display retry time frame with no tasks loaded and a logged in user
  - Rate limiting cannot be triggered by logged in user
- should display an error modal when fetching issue previews fails on page load
  - API call has been removed

New tests:
- Main page shouldn't display invalid issue

### CODE REVIEW

**Comment by 0x4007** [comment]
There's one failing still

**Comment by nick-dev123** [comment]
@0x4007 Fixed the failing test.

**Review by rndquu — CHANGES_REQUESTED** [review]
Screenshot 2024-10-11 at 12 28 08

CSS is broken, pls fix

**Review by 0x4007 — APPROVED** [review]
Looks fine on CD and CI is green.

**Comment by gentlementlegen** [comment]
I do not know if that's introduced by this pull request or because I test locally, but when I log in I have this in my app bar
image

I do not see this on the production version.

---

I reverted the changes and tested, the issue does not happen so I believe this fix introduces that.

**Comment by nick-dev123** [comment]
@gentlementlegen The lock icon is called `augment-access-button` and as far as I can tell it allows users to access private repositories related tasks.
The `Should display filters on small devices` test case had the following requirement:
`cy.get("#augment-access-button").should("be.visible");`
I can remove it if it's not required.

**Comment by gentlementlegen** [comment]
@nick-dev123 I meant that the "login button" should not be there, as I am logged in. Do you have this happening as well?

**Comment by rndquu** [comment]
> @nick-dev123 I meant that the "login button" should not be there, as I am logged in. Do you have this happening as well?

Can't not reproduce, the login button is hidden when I'm logged in.

@nick-dev123 Could you merge the latest `development` branch?

**Comment by nick-dev123** [comment]
@rndquu done

File: src/home/rendering/display-github-user-information.ts [hunk 1]
```diff
@@ -37,12 +37,14 @@ export async function displayGitHubUserInformation(gitHubUser: GitHubUser) {
     window.location.reload();
   });
 
+  containerDivElement.appendChild(authenticatedDivElement);
```
Comment (0x4007) [inline src/home/rendering/display-github-user-information.ts:40]: Is this correct to append? I expected only changes to tests.

Comment (rndquu) [reply src/home/rendering/display-github-user-information.ts:40]: @nick-dev123 Why was the UI updated?

**Pull request closed without merging**